torch/csrc/jit/runtime/profiling_record.cpp

```diff
@@ -61,16 +64,30 @@ void ProfilingRecord::insertShapeProfile(Node* n, Value* i) {
   n->replaceInputWith(i, pn->output());
 }
 
+bool isProfilable(Node* n) {
+
+  if (IsFusibleOperation(n) || isSupported(n)) {
```

Comment:
AFAIK, `isSupported` is not exposed outside of `tensorexpr_fuser.cpp`. How does this code work?

Reply:
I made it public need to push that change as well